SConstruct

```diff
@@ -298,12 +298,15 @@ if arch == "Darwin":
   qt_env["FRAMEWORKS"] += [f"Qt{m}" for m in qt_modules] + ["OpenGL"]
   qt_env.AppendENVPath('PATH', os.path.join(qt_env['QTDIR'], "bin"))
 else:
-  qt_env['QTDIR'] = "/usr"
+  qt_query = os.popen('qmake -query').read()
+  qt_query = {kv[0]: kv[1] for kv in [line.split(':') for line in qt_query.split('\n')] if len(kv) == 2}
```

Comment:
can you split this up so it's not two for loops on one line? can also just extract QT_INSTALL_HEADERS if that's simpler